### PR TITLE
Implementing land state and refill state

### LINKED ISSUE #171: Add refueling state
# Refueling State

## Problem
Add a refueling state that goes back to the refuel station and allows us to reload bottles and replace batteries.

## Solution
Make a new state file and implement it :) Look at the other state files for inspiration

### LINKED ISSUE #172: Implement Land state
# Land state

## Problem

We need the land state to actually go back to where it launches from, or where we are allowed to land for the competition.

## Solution

Just implement what I said above in state_machine/states/impl/land_impl.py :)

## Additional Information

Provide any additional information.

### PR DESCRIPTION
**Pull request by DJDMVP**
Adding the correct state logic for land state and transitions to land state.

### CODE REVIEW

File: state_machine/states/impl/airdrop_impl.py [hunk 1]
```diff
@@ -60,9 +65,15 @@ async def run(self: Airdrop) -> State:
         else:
             self.drone.servo_num = self.drone.servo_num + 1
 
-        if self.drone.bottle_num == 6:
-            return Land(self.drone, self.flight_settings)
-        return Waypoint(self.drone, self.flight_settings)
+        continuerun: bool = False
```
Comment (cjhr95) [inline state_machine/states/impl/airdrop_impl.py]: Snake case plz

File: state_machine/states/impl/airdrop_impl.py [hunk 2]
```diff
@@ -60,9 +65,15 @@ async def run(self: Airdrop) -> State:
...
+
+        for cylinder in cylinders:
+            if cylinder["Loaded"]:
+                continuerun = True
+
+        if continuerun:
```
Comment (cjhr95) [inline state_machine/states/impl/airdrop_impl.py]: `continuerun` needs snake case

File: state_machine/states/impl/airdrop_impl.py
Comment (cjhr95) [inline state_machine/states/impl/airdrop_impl.py:1]: Edit your logging so it states which bottle number is being dropped

**Review by Ooglely — DISMISSED** [review]
The json data isn't typed when it's read but if Connor is good with it I'm good with it

**Review by cjhr95 — CHANGES_REQUESTED** [review]
Little stuff for a big guy

**Review by cjhr95 — APPROVED** [review]
I prefer f-strings, good stuff

**Pull request merged**